tests/foreman/ui/test_organization.py

```diff
@@ -407,498 +380,6 @@ def test_negative_update(self):
                         common_locators['name_haserror'])
                     self.assertIsNotNone(error)
 
-    @run_only_on('sat')
-    @tier2
-    def test_positive_create_with_domain(self):
```

Comment:
removing, not porting all tests related to create with entity functionality as it is actually impossible to create organization with entity right away as you moved to edit screen once you specified name for new organization